examples/delegator/accumulator/lib.rs

```diff
@@ -17,30 +17,30 @@
 pub use self::accumulator::Accumulator;
 use ink_lang as ink;
 
-#[ink::contract(version = "0.1.0")]
-mod accumulator {
-    /// Holds a simple i32 value that can be incremented and decremented.
+#[ink::contract]
+pub mod accumulator {
+    // / Holds a simple i32 value that can be incremented and decremented.
```

Comment:
```suggestion
    /// Holds a simple i32 value that can be incremented and decremented.
```

Reply:
I see there are way more occurrences of this `// /`. Is this intentional?

Reply:
It is intentional since the doc comments screw up `cargo expand` output magically.

Reply:
Mhm, do I understand you right that the only purpose of this hack is to work around a bug in `cargo expand`?

Reply:
It is rather a bug in the Rust compiler from what I know since it is having some hiccups with internal attributes. (Everything that is a `#![...]`).

Reply:
Ok, I see. Yeah I think it's rather ugly and goes against our "as close to Rust as possible" mantra. Hmm, not sure how to handle this. We could maybe track it in a follow-up issue?

Reply:
We actually cannot fix this. This needs to be fixed in the Rust compiler.